ush/parsing_namelists_FV3.sh

```diff
@@ -80,674 +80,419 @@ else
-  lw_file_clouds     = ${lw_file_clouds:-'rrtmgp-cloud-optics-coeffs-lw.nc'}
-  sw_file_gas        = ${sw_file_gas:-'rrtmgp-data-sw-g112-210809.nc'}
-  sw_file_clouds     = ${sw_file_clouds:-'rrtmgp-cloud-optics-coeffs-sw.nc'}
-  rrtmgp_nGptsSW     = ${rrtmgp_nGptsSW:-"112"}
-  rrtmgp_nGptsLW     = ${rrtmgp_nGptsLW:-"128"}
-  rrtmgp_nBandsLW    = ${rrtmgp_nBandsLW:-"16"}
-  rrtmgp_nBandsSW    = ${rrtmgp_nBandsSW:-"14"}
-  doGP_cldoptics_LUT = ${doGP_cldoptics_LUT:-".false."}
-  doGP_lwscat        = ${doGP_lwscat:-".false."}
-EOF
-
+# Set variables for global_control.nml template
+
+local BLOCKSIZE=${blocksize}
+local CHKSUM_DEBUG=${chksum_debug}
+local DYCORE_ONLY=${dycore_only}
+local CCPP_SUITE=${CCPP_SUITE}
+
+local MAX_OUTPUT_FIELDS=300
+
+local DOMAINS_STACK_SIZE=${domains_stack_size:-3000000}
+local PRINT_MEMORY_USAGE=${print_memory_usage:-".false."}
+
+local INPES=${layout_x}
+local JNPES=${layout_y}
+local IO_LAYOUT=${io_layout}
+local NPX=${npx}
+local NPY=${npy}
+local NTILES=${ntiles}
+local NPZ=${npz}
+local DZ_MIN=${dz_min:-"6"}
+local PSM_BC=${psm_bc:-"0"}
+local MAKE_NH=${make_nh}
+local FV_DEBUG=${fv_debug:-".false."}
+local RANGE_WARN=${range_warn:-".true."}
+local N_SPONGE=${n_sponge:-"10"}
+local NUDGE_QV=${nudge_qv:-".false."}
+local NUDGE_DZ=${nudge_dz:-".false."}
+local TAU=${tau:-10.}
+local FAST_TAU_W_SEC=${fast_tau_w_sec:-"0.2"}
+local RF_CUTOFF=${rf_cutoff:-"7.5e2"}
+local D2_BG_K1=${d2_bg_k1:-"0.15"}
+local D2_BG_K2=${d2_bg_k2:-"0.02"}
+local KORD_TM=${kord_tm:-"-9"}
+local KORD_MT=${kord_mt:-"9"}
+local KORD_WZ=${kord_wz:-"9"}
+local KORD_TR=${kord_tr:-"9"}
+local HYDROSTATIC=${hydrostatic}
+local PHYS_HYDROSTATIC=${phys_hydrostatic}
+local USE_HYDRO_PRESSURE=${use_hydro_pressure}
+local UPDATE_FULL_OMEGA=${pass_full_omega_to_physics_in_non_hydrostatic_mode:-".false."}
+local K_SPLIT=${k_split}
+local N_SPLIT=${n_split}
+local NWAT=${nwat:-2}
+local NA_INIT=${na_init}
+local DNATS=${dnats}
+local FV_SG_ADJ=${fv_sg_adj:-"450"}
+local NORD=${nord:-3}
+local DDDMP=${dddmp:-0.1}
+local D4_BG=${d4_bg:-0.15}
+local VTDM4=${vtdm4}
+local DELT_MAX=${delt_max:-"0.002"}
+local DO_VORT_DAMP=${do_vort_damp}
+local EXTERNAL_IC=${external_ic}
+local EXTERNAL_ETA=${external_eta:-.true.}
+local GFS_PHIL=${gfs_phil:-".false."}
+local NGGPS_IC=${nggps_ic}
+local MOUNTAIN=${mountain}
+local NCEP_IC=${ncep_ic}
+local D_CON=${d_con}
+local HORD_MT=${hord_mt}
+local HORD_VT=${hord_xx}
+local HORD_TM=${hord_xx}
+local HORD_DP=${hord_dp}
+local HORD_TR=${hord_tr:-"8"}
+local ADJUST_DRY_MASS=${adjust_dry_mass:-".true."}
+local DRY_MASS=${dry_mass:-98320.0}
+local CONSV_TE=${consv_te}
+local DO_SAT_ADJ=${do_sat_adj:-".false."}
+local PRINT_FREQ=${print_freq}
+local WARM_START=${warm_start}
+local NO_DYCORE=${no_dycore}
+local AGRID_VEL_RST=${agrid_vel_rst:-".true."}
+local READ_INCREMENT=${read_increment}
+local RES_LATLON_DYNAMICS=${res_latlon_dynamics}
+local ATM_IGNORE_RST_CKSUM=.false.
+local INCREMENT_FILE_ON_NATIVE_GRID=.false.
```

Comment:
I think `INCREMENT_FILE_ON_NATIVE_GRID` needs to first check if `increment_file_on_native_grid` is set.  This is set `.true.` when running JEDI DA.

```suggestion
local INCREMENT_FILE_ON_NATIVE_GRID=${increment_file_on_native_grid:-.false.}
```